content/swift/concepts/protocols/terms/equatable/equatable.md

```diff
@@ -0,0 +1,124 @@
+---
+Title: 'Equatable'
+Description: 'Enables two instances of a type to be compared for equality.'
+Subjects:
+  - 'iOS'
+  - 'Mobile Development'
+Tags:
+  - 'Protocols'
+  - 'Equality'
+CatalogContent:
+  - 'learn-swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+**`Equatable`** is a protocol that enables two instances of a type to be compared for equality using the `==` or `!=` operators.
+
+## Syntax
+
+In a straightforward case, for a type to conform to `Equatable`, all of its properties must also conform to `Equatable`:
+
+```pseudo
+struct myStruct: Equatable {
+  // properties that conform to Equatable
+}
+```
+
+When checking for equality between two instances of the `myStruct` [structure](https://www.codecademy.com/resources/docs/swift/structures), all properties will be compared.
+
+### Syntax using a custom `==` method
+
+If a customised approach is needed for equality comparison, then a custom `==` method can be included so that the type can conform to `Equatable`:
+
+```pseudo
+struct myStruct: Equatable {
+  // properties
+
+  static func == (lhs: myStruct, rhs: myStruct) -> Bool {
+    // custom logic for equality comparison
+  }
+}
+```
+
+A custom `==` method can be helpful if only specific properties need to be compared, or in cases when some of the properties don't conform to `Equatable`.
+
+## Example
+
+In the example below, a `TeaCup` structure is declared with three properties `color`, `volume`, and `unit`. These properties all conform to the `Equatable` protocol so instances of `TeaCup` can be compared using the `==` operator.
+
+Below the `TeaCup` structure, two instances of `TeaCup` are created and compared for equality:
+
+```swift
```

Comment:
looks like this code is using tabs, indentations should be done with 2 spaces. Please update both examples

Reply:
@caupolicandiaz Thanks so much for the review! I updated the code examples to have 2 spaces and also committed the other suggestions.